webpack.config.js

```diff
@@ -13,17 +13,26 @@ module.exports = [
     },
     mode: 'production',
     plugins: [],
-    stats: 'errors-only'
+    stats: 'errors-only',
+    optimization: {
+      minimize: true
+    }
   },
   {
     target: 'web',
-    entry: './node_modules/google-protobuf/google-protobuf.js',
+    entry: [
+      './node_modules/google-protobuf/google/protobuf/wrappers_pb.js',
```

Comment:
Are the wrappers really need to be included this way? The usual way to load extensions like this is to add them to the `require` array in the config (https://github.com/gonicus/protoc-gen-qx/blob/master/config.js#L27) did you try that?